src/pages/patientView/patientHeader/SignificantMutationalSignatures.tsx

```diff
@@ -0,0 +1,143 @@
+
+        const significantMutationalSignaturesForSample: ISignificantMutationalSignaturesForSample = prepareMutationalSignaturesForHeader(mutationalSignatureData, mutationalSignatureMetaData, uniqueSampleKey);
+        let hasMutationalSignatures: boolean = true;
+
+        if(_.isEmpty(significantMutationalSignaturesForSample.significantSignatures)){
+            hasMutationalSignatures = false;
+        }
+
+        return (
+            <div style={{maxWidth:250}}>
+                {/*<h6 style={{textAlign:"center"}}>*/}
+                    {/*Mutational Signature Confidences*/}
+                {/*</h6>*/}
+                <span className="small">
+                    {significantMutationalSignaturesForSample.confidenceStatement}
+                </span>
+                <hr style={{marginTop:10,marginBottom:10}}/>
+                {hasMutationalSignatures &&
+                    <table>
+                    <th>Significant Mutational Signatures</th>
+                        <th>Exposure</th>
+                    {Object.keys(significantMutationalSignaturesForSample.significantSignatures).map((significantSignatureValue)=>
+                        <tr><td style={{paddingTop: 3}}>{significantSignatureValue}</td>
+                        <td style={{paddingTop: 3}}>{this.progressBar(significantMutationalSignaturesForSample.significantSignatures[significantSignatureValue].toString(), false)}</td>
+                        </tr>)}
+                    </table>
+                }
+
+            </div>
+        );
+    }
+
+    public progressBar(confidence:string, determineColor:boolean) {
+        let confidencePerc = Math.round(parseFloat(confidence) * 100);
+
+        // Coloring same as CVR as discussed with @anoopbr
+        let progressBarClassName:string = "progress-bar-info";
+        let progressBarStyle:{ [s:string]:string; } = {"backgroundColor":"darkgray"};
+
+        return (
+            <div className="progress" style={{position:"relative",width:100,marginBottom:0}}>
+                <div data-test="progress-bar" className={`progress-bar ${progressBarClassName}`}
+                     role="progressbar" aria-valuenow={`${confidencePerc}`}
+                     aria-valuemin="0" aria-valuemax="100"
+                     style={Object.assign(progressBarStyle, {width:`${confidencePerc}%`})} />
+                <div style={{position:"absolute",
+                    textShadow:"-1px 0 white, 0 1px white, 1px 0 white, 0 -1px white",
+                    width:100,
+                    marginTop:2,
+                    textAlign:"center"}}
+                >{confidencePerc}%</div>
+            </div>
+        );
+    }
+
+}
+
+export function prepareMutationalSignaturesForHeader(mutationalSignatureData: IMutationalSignature[], mutationalSignatureMetaData: IMutationalSignatureMeta[],
+                                                     uniqueSampleKey: string): ISignificantMutationalSignaturesForSample{
```

Comment:
the way we've been organizing code in the frontend is to move stuff like this into `MutationalSignaturesUtils`

Reply:
Ok, sounds good! I can move the equivalent function for the mutational signature table in there as well for organization.